Gemfile

```diff
@@ -42,6 +42,7 @@ gem 'faraday', require: nil
 gem 'graphql', require: nil
 gem 'google-protobuf', '< 3.12' if !defined?(JRUBY_VERSION) && RUBY_VERSION < '2.5'
 gem 'grpc' if !defined?(JRUBY_VERSION)
+gem 'json'
```

Comment:
Adding this to avoid `warning` spam in the console when multiple json gems are present.